src/Frame.cc

```diff
@@ -1397,6 +1397,35 @@ Frame::recalcResizeDrag(int nx, int ny, bool left, bool top)
 	_gm.y = top ? (_click_y - _gm.height) : _click_y;
 }
 
+void
+Frame::moveToHead(const std::string& arg)
+{
+	int head_nr;
+	try {
+		head_nr = std::stoi(arg);
+	} catch (std::invalid_argument&) {
+		head_nr = -1;
+	}
+
+	if (head_nr != -1) {
+		// valid value from stoi above
+	} else if (StringUtil::ascii_ncase_equal(arg, "LEFT")) {
```

Comment:
Not sure if you like a flatter code flow like this, or move this if/else chain inside the catch block.